fjord/base/tests/test_google_utils.py

```diff
@@ -33,8 +35,9 @@ def handler(*args, **kwargs):
 
 
 class TestGoogleUtils(TestCase):
-    @set_sys_argv(['test'])
-    def test_ga_track_event_test(self):
+    @override_settings(DEBUG=True)
+    @set_sys_argv([])
```

Comment:
This is... weird. I can't think of a better way to do it, but munging `sys.argv` seems nasty.